XamlControlsGallery/ControlPages/ComboBoxPage.xaml

```diff
@@ -44,7 +44,7 @@
             <local:ControlExample.Example>
                 <StackPanel>
                     <ComboBox x:Name="Combo2" ItemsSource="{x:Bind Fonts}" DisplayMemberPath="Item1"
-                            SelectedValuePath="Item2" Header="Font" Width="200" Loaded="Combo2_Loaded" />
+                            SelectedValuePath="Item2" Header="Font" Width="300" Loaded="Combo2_Loaded" />
```

Comment:
```suggestion
                            SelectedValuePath="Item2" Header="Font" MinWidth="200" Loaded="Combo2_Loaded" />
```

Does the above change also fix the issue? I'm wondering if swapping one hardcoded value for another is the way to go.

Reply:
Yes this also fixes this. Setting the MinWidth is probably the better way to go here.